src/cli/domain/get-missing-deps.js

```diff
@@ -22,7 +22,7 @@ module.exports = function(dependencies, components){
         delete require.cache[pathToModule];
       }
 
-      var required = require(pathToModule);
+      require(pathToModule);
```

Comment:
The easiest way to know if you can require something is to wrap a require into a try/catch. Given the only value of this is to ensure this doesn't throw, no need to assign the value to a "required" variable (which, in fact, the linter finds as unused)